reference/4.0/Microsoft.PowerShell.Core/About/about_Script_Blocks.md

```diff
@@ -138,21 +138,44 @@ For more information about the call operator, see [about_Operators](about_Operat
 
 ## Using delay-bind script blocks with parameters
 
-A **delay-bind** script block allows you to pipe input to a given parameter,
-then use script blocks for other parameters using the pipeline variable `$_` to
-reference the same object.
+A **delay-bind** script block allows access to the pipeline input object using
```

Comment:
How does it "allow access"? I think you need to flip that around. The way it is stated in the NOTE block of the other articles sounds best to me.

A typed parameter that accepts pipeline input (by Value) or (by PropertyName) enables use of delay-bind script blocks on the parameter. The delay-bind script block is run automatically during ParameterBinding. The result is bound to the parameter.